code/modules/vchat/css/ss13styles.css

```diff
@@ -172,6 +172,7 @@ h1.alert, h2.alert		{color: #000000;}
 .say_quote				{font-family: Georgia, Verdana, sans-serif;}
 .terminus				{font-family: "Times New Roman", Times, serif, sans-serif}
 .interface				{color: #330033;}
+.spacer					{color: #9c660b;}
```

Comment:
Can you add a Vorestation add comment to this?

Reply:
Added.